src/pages/Home/ReceiveModal/ReceiveModal.tsx

```diff
@@ -95,87 +94,75 @@ const ReceiveModal: FC<Props> = ({ onClose }) => {
   const {
     register,
     handleSubmit,
-    formState: { errors, isValid, submitCount },
+    formState: { errors },
   } = useForm<IFormInputs>({
     mode: "onChange",
   });
 
   const onSubmit: SubmitHandler<IFormInputs> = (_data) =>
     generateInvoiceHandler();
 
-  return createPortal(
-    <ModalDialog close={onClose}>
-      <div className="text-xl font-bold">
-        {showLnInvoice ? t("wallet.create_invoice_ln") : t("wallet.fund")}
-      </div>
+  const onBack = () => setAddress("");
 
+  return createPortal(
```

Comment:
the transition doesn't really work with a portal. need to google how to make it work